content/workspace/openbb-copilot.md

```diff
@@ -12,336 +12,616 @@ keywords:
-- Interact with your uploaded files
-- Automatically use data from the dashboard
-- Automatically retrieve data using the OpenBB API or any custom backend you have added
-- Create charts, tables, and other text artifacts, like summaries
-- Iteratively manipulate and transform data and charts
-- Be used to search the web for additional information
-- Provide citations
+Built specifically for financial professionals, the Copilot transforms traditional workflows by eliminating manual data retrieval, accelerating analysis, and providing context-aware insights based on your specific datasets and workspace configuration.
 
-## Introduction
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/0a5e0075-7c50-44cb-8d45-2be1817deab5.png"
+    width="100%"
+  />
+</div>
+
+## Basics
 
-To use OpenBB Copilot, click on the purple icon located at the center right of the
-Workspace screen:
+### Structure
 
-<div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/open_copilot.jpg" alt="open openbb copilot" width="70%" height="70%" />
+<div
+  style={{ display: "flex", justifyContent: "center", alignItems: "center" }}
+>
+  <img
+    src="https://openbb-cms.directus.app/assets/d7efdb97-fecd-4d58-8094-84fa82bf4bd4.png"
+    width="40%"
+  />
 </div>
 
-Under most circumstances, you can treat OpenBB Copilot like a personal research
-assistant. OpenBB Copilot has access to the widgets (including your uploaded
-files) on your currently-active dashboard, the OpenBB API and any custom backend
-endpoints you have added to OpenBB Workspace. 
+The Copilot interface is designed for intuitive interaction with a clean, three-section layout that maximizes usability and clarity.
+
+- **Header:** Contains essential controls for conversation management, including the current conversation the user is in. But also the capability to start a new chat, clear the current chat history, expand to full AI-mode or hide the agent.
+
+- **Body:** The primary interaction area where conversations unfold. This section displays the Copilot's responses, step-by-step reasoning processes, generated artifacts (charts, tables, code), and any intermediate results. The body automatically scrolls and organizes content chronologically, making it easy to follow complex multi-step analyses. When hovering on messages it also displays the datetime associated with these.
+
+- **Footer:** This is where the data and widget management occurs for what will be used as context for the agent. This is also where the user can add their own custom AI agents or access their prompt library.
+
+### Full AI-chat mode
 
-By utilizing all of these sources of data, OpenBB Copilot can assist you in
-performing a wide variety of research and analysis tasks.
+The Copilot panel offers adaptive sizing to match your workflow needs.
```

Comment:
This can be said simpler like "you can resize the panel by dragging the border" or similar